calico/getting-started/kubernetes/windows-calico/limitations.mdx

```diff
@@ -12,6 +12,7 @@ description: Review limitations before starting installation.
 | Install and upgrade | **Supported**: - Manifest with manual upgrade (deprecated)<br />- Operator install<br /><br />**Not supported**: <br />- Non-cluster hosts<br />- Typha component for scaling (Linux-based feature)                                                                                                                                                                                                                                                                                                         |
 | Networking          | **Supported**:<br />- {{prodname}} VXLAN, no cross-subnet or VXLAN MTU settings with [limitations](#{{prodname}}-vxlan-networking-limitations)<br />- {{prodname}} non-overlay mode with BGP peering with [limitations](#{{prodname}}-bgp-networking-limitations)<br />- IPv4<br /><br />**Not supported**: <br />- Overlay mode with BGP peering<br />- IP in IP overlay with BGP routing<br />- Cross-subnet support and MTU setting for VXLAN<br />- IPv6 and dual stack<br />- Service advertisement |
 | Security            | **Not supported**: <br />- Application Layer Policy (ALP) for Istio<br />- Policy for hosts (host endpoints, including automatic host endpoints)<br />- Encryption with WireGuard                                                                                                                                                                                                                                                                                                            |
+| Visibility and troubleshooting | **Not supported**:<br/>- Packet capture<br />- DNS logs<br />- iptable logs<br />- L7 logs                                                                                                                                                                                      |
```

Comment:
OS doesn't have DNS logging/L7 logging in general so Windows is no exception.

I'd express "iptables logs" as "'Log' action in network policy"

Reply:
Thanks, changes made.